chaoslib/litmus/platform/gke/DaemonSet.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: cpu-hog
+spec:
+  template:
+    metadata:
+      labels:
+        app: cpu-hog
+      name: cpu-hog
+    spec:
+      containers:
+      - image: uditgaurav/cpu:v1
```

Comment:
Ideally, this image should be `litmuschaos/cpu:latest` -> or some similar. You can submit the dockerfile here https://github.com/litmuschaos/test-tools

(refer to a PR on that repo to setup CI/CD for this image). As a first step, can we build this image manually and replace name here?